src/UseEffect/AntiPatterns/Examples/ImperativeChange.tsx

```diff
@@ -22,7 +18,9 @@ export function ImperativeChangeAntiPattern(props: Props) {
           id="inputValue"
           value={inputValue}
           onChange={(event) => {
-            setInputValue(event.target.value);
+            const { value } = event.target;
+            setInputValue(value);
+            setLocalFirstName(`${value} 1`);
```

Comment:
This demonstrates that we should not rely on `useEffect` to do imperative things. Simply do things in the imperative call directly.